### DISCUSSION TITLE
2023-07-19

### DISCUSSION THREAD

**Discussion by jaehee329** [woowacourse-teams/2023-haru-study#100] [daily_meeting]
### 오늘의 할 일
- 백엔드
오전 중으로 스터디 조회 기능 PR merge
스터디 진행 기능 완성
스터디 참여 기능 완성
### 내일의 할 일
기능 추가 및 목요일까지 완성된 기능들을 배포
발표자들은 나머지와 내용 협의해서 발표 준비
### 언젠가 해야할 일
기획 변경된 이유 정리
DNS 어떻게 할지
UX 저자 워크숍 사전 미션(~7/25 화) -> 준이 제공한 내용 참고해 데모 끝나고 월요일에 하기로 결정

**Comment by woo-jk** [comment]
### 체크인
9점 - 아침부터 배고파서 -1

**Comment by nlom0218** [comment]
### 체크인

5점 - 아까운 내 16000원

### 체크아웃

10점 - 재밌는 페이지 개발

**Reply by woosung1223** [reply]
택시기사 아저씨: 아침부터 개꿀 ㅋ

**Reply by nlom0218** [reply]
왠지 바로 잡히더라

**Reply by aak2075** [reply]
ㅠㅠ

**Comment by jaehee329** [comment]
### 체크인
6점 - 날마다 아침에 일어나기 힘들어지고 있어요
내일 예비군이라 짐 다 들고가야돼서 점수 감점

**Reply by woosung1223** [reply]
thank for your duty.

**Reply by aak2075** [reply]
모디의 모다기...

**Comment by woosung1223** [comment]
### 체크인
8점 - 잠은 많이 잤는데 에어컨 키고 자서 그런가 목이 칼칼하네용 -2점

### 체크아웃
8점 - 할 일을 어느정도 다 한 것 같아서 기분이 나쁘지 않네요 🥸

**Comment by MoonJeWoong** [comment]
### 체크인
10점 - 아침부터 팀원들에게 재밌는 일들이 많이 일어났네요! 웃다보니 기분 좋아져서 10점입니다 😄 

### 체크아웃
10점 - 계획했던 개발 사항들을 모두 마무리했네요! 너무 고생들하셨습니다~

**Comment by yeopto** [comment]
### 체크인

6점 - 어제 불태워서 너무 피곤합니다. 제 주제를 알고 다시는 패턴에 도전하지 않겠습니다.

**Comment by aak2075** [comment]
### 체크인

9점 - 잘 잤지만 심장이 부담을 느끼는 것 같기도 하네요 화이팅 합시다
